channels/pkg/channels/channel_version.go

```diff
@@ -39,6 +40,17 @@ type ChannelVersion struct {
 	Channel *string `json:"channel,omitempty"`
 }
 
+func stringValue(s *string) string {
+	if s == nil {
+		return ""
+	}
+	return *s
+}
+
+func (c *ChannelVersion) String() {
```

Comment:
@justinsb out of curiosity, what's the need for reference/pointer string `*string` inside of a struct?  By default behavior of a string, the initial value is a blank string, not a nil and you wouldn't need a helper function to do what's already a functionality of the language. The *ChannelVersion is already a copy of the address block you allocated. I'm trying to wrap my head around things.